timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -15,35 +15,18 @@ limitations under the License.
 -->
 <template>
   <v-container fluid>
-    <!-- Right side menu for selected events -->
-    <v-navigation-drawer
-      v-model="selectedEvents.length"
-      fixed
-      right
-      width="600"
-      style="box-shadow: 0 10px 15px -3px #888"
-    >
+    <!-- Right side menu -->
+    <v-navigation-drawer v-if="showRightSidePanel" fixed right width="600" style="box-shadow: 0 10px 15px -3px #888">
       <template v-slot:prepend>
         <v-toolbar flat>
-          <v-toolbar-title>{{ selectedEvents.length }} events selected</v-toolbar-title>
-
+          <v-toolbar-title>Right Side Panel</v-toolbar-title>
           <v-spacer></v-spacer>
-
-          <v-btn icon @click="selectedEvents = []">
+          <v-btn icon @click="showRightSidePanel = false">
             <v-icon>mdi-close</v-icon>
           </v-btn>
         </v-toolbar>
       </template>
-
-      <v-container>
-        <v-list>
-          <v-list-item v-for="(event, index) in selectedEvents" :key="index">
-            <v-list-item-content>
-              <v-list-item-title>{{ event._source.message }}</v-list-item-title>
-            </v-list-item-content>
-          </v-list-item>
-        </v-list>
-      </v-container>
+      <v-container> Content go here </v-container>
```

Comment:
Is this a place holder? Should this be included in this PR?

Reply:
Good catch - yeah this is a placeholder for now because the content was refactored in this pane. I added a comment to make this clear.

We will use it in a few PRs from now.